migrations/2018_01_18_130400_rename_users_groups_to_groups_users.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * (c) Toby Zerner <toby.zerner@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Flarum\Database\Migration;
+
+return Migration::renameTable('users_groups', 'group_user');
```

Comment:
The table should be called group**s**_user**s**

Reply:
Seems like that's not possible:

https://laravel.com/docs/5.6/eloquent-relationships#many-to-many

Reply:
disregard this, as per our discussion on Discord we are changing our convention to match Laravel's :)